composer/algorithms/factorize/factorize.py

```diff
@@ -47,8 +47,8 @@ def apply_factorization(model: torch.nn.Module,
             few channels are unlikely to be accelerated by factorization due
             to poor hardware utilization. Default: ``512``.
         latent_channels (int or float, optional): number of latent channels to use in factorized
-            convolutions. Can be specified as either an integer > 1 or as
-            float within [0, 1). In the latter case, the value is
+            convolutions. Can be specified as either an integer > 1 or as a
+            float within ``[0, 1)``. In the latter case, the value is
```

Comment:
for intervals, I'm pretty okay with anything as long as we're consistent, but I'm pretty sure I've seen torch write these as just raw text. Also as $$math$$, at least in [rand](https://pytorch.org/docs/stable/generated/torch.rand.html).

Just highlighting this as a thing we need to be careful to be consistent about; feel free to just resolve this comment.

Reply:
So should I change them to :math:? I've been consistently changing them to inline code, but I'm also fine with whatever.